package.json

```diff
@@ -13,7 +13,7 @@
     "bootstrap": "lerna bootstrap --force-local",
     "bootstrap-no-build": "lerna bootstrap --force-local --ignore-scripts",
     "update": "lerna publish --skip-git --skip-npm --exact",
-    "publish": "lerna publish --skip-git --repo-version $VERSION --yes --exact",
+    "publish": "lerna publish --skip-git --yes --exact",
```

Comment:
This has been breaking deployments and is in for backporting. I need to figure out what to do about that.